content/src/service/access/AccessCheckerForWearables.ts

```diff
@@ -19,47 +18,27 @@ export class AccessCheckerForWearables {
     }
 
     const pointer: Pointer = pointers[0].toLowerCase()
+    const parsed = await parseUrn<{ contractAddress: string; id: string; url: URL }>(pointer)
+    if (parsed) {
```

Comment:
should the type of the parsed URN be taken into account? to avoid future complications with new format additions to `parseUrn`

Reply:
Now that I'm using the new lib version, I'm expecting `parsed` to be  `BlockchainCollectionV2Asset | null`. Is that what you are asking?

Reply:
Yep, the static type assertion gives you no guarantees tho, you may also add a `parsed.type == 'blockchain-collection-v2'` condition somewhere. https://github.com/decentraland/urn-resolver/blob/main/src/types.ts#L73